tools/gen_addon_readme.template

```diff
@@ -0,0 +1,86 @@
+{%- macro fragment(name, title, sub='=') %}
+{%- if name in fragments %}
+{{- title }}
+{{ sub * title|length }}
+
+{{ fragments[name] }}
+{% endif %}
+{%- endmacro -%}
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+{% for (image, target, alt) in badges -%}
+.. |badge{{ loop.index }}| image:: {{ image }}
+    :target: {{ target or '#' }}
+    {%- if alt %}
+    :alt: {{ alt }}
+    {%- endif %}
+{% endfor %}
+{% for _ in badges %}|badge{{ loop.index }}| {% endfor %}
+
+{{ '=' * manifest.name|length }}
+{{ manifest.name }}
+{{ '=' * manifest.name|length }}
+
+{{ fragments.get('DESCRIPTION', '') }}
+.. contents::
+   :local:
+
+{{ fragment('INSTALL', 'Installation') }}
+{{- fragment('CONFIGURE', 'Configuration') }}
+{{- fragment('USAGE', 'Usage') }}
+{{- fragment('DEVELOP', 'Development') }}
+{{- fragment('ROADMAP', 'Known issues / Roadmap') -}}
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{{ repo_name }}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Do not contact contributors directly about support or help with technical issues.
```

Comment:
Probably out of scope for this PR, but it would be nice to add some actionable advice here. How about: "Either open an Issue on the Github project, or post your query to the OCA Contributors mailing list.". Not sure if we have a link to the Github project anywhere in the readme; we surley should.

Reply:
The repo link is 4 lines above.